developers/weaviate/concepts/indexing/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Indexing
+sidebar_position: 1
+image: og/docs/indexing.jpg
+# tags: ['basics']['indexes']
+---
+
+This section of the documentation discusses indexing concepts as they relate to Weaviate.
+
+For configuration examples without additional contextual material, see the [Vector index](/developers/weaviate/configuration/indexing-vector) and [Inverted index](/developers/weaviate/configuration/inverted-indexes) configuration pages.
+
+## Vector indexes
+
+Vector embeddings encapsulate meaning for the objects that they represent. Vector indexes make it easier to efficiently search large vector spaces in order to find objects with similar vector embeddings.
+
+Weaviate offers these vector index types:
+
+- [Hierarchical Navigable Small World (HNSW) indexes](/developers/weaviate/concepts/indexing/hnsw-indexes)
+- [Flat indexes](/developers/weaviate/concepts/indexing/flat-indexes)
+- [Dynamic indexes](/developers/weaviate/configuration/indexing-vector/dynamic-indexes)
+
+## Inverted indexes
+
+Inverted indexes, also known as keyword indexes, make textual and numeric searches more efficient. Inverted indexes use the [BM25](/developers/weaviate/concepts/indexing/inverted-indexes#bm25-ranking-algorithm) ranking algorithm.
```

Comment:
Couple of suggestions:

- I would not use "keyword indexes" here as the broader term, because the range filter for example is numeric. 
- > Inverted indexes use the [BM25]
    - I think this is the other way around. BM25 algos use 'indexSearchable' to speed up the score calculations.